docs/topics/development/dependencies.md

```diff
@@ -0,0 +1,62 @@
+# Project Dependencies
+
+This document describes how to add/upgrade dependencies in the project.
+We use pip to manage dependencies and hashin to lock versions. We use npm to manage frontend dependencies.
+
+## Python
+
+### Adding Python Dependencies
+
+We have 2 requirements files for python dependencies:
+
+- prod.txt
+- dev.txt
+
+Prod dependencies are used by our django app in runtime.
+They are strictly required to be installed in the production environment.
+
+Dev dependencies are used by our django app in development or by tools we use for linting, testing, etc.
+
+To ensure a standard and sane process for installing packages,
+use the `make install` command to install a new package.
+This will add your specified version to one of our requirements/(dev|prod).txt files and
+install the package in your local container.
+
+```bash
+make install [package]==[version] --dev --prod
```

Comment:
😕 where is this make recipe defined?

Reply:
It's not. I realized that now, I was imagining this would be good to have cause I needed to install several packages while working on this and I never remember how to do it.

But then I moved on and didn't actually implement it. I'll remove for now and keep the older manual process.